webknossos-datastore/app/com/scalableminds/webknossos/datastore/image/ImageCreator.scala

```diff
@@ -116,27 +118,25 @@ object ImageCreator extends LazyLogging {
         if (isSegmentation)
           idToRGB(b(idx))
         else {
-          val rF = color.map(_.r).getOrElse(1d) // rF = red factor
-          val gF = color.map(_.g).getOrElse(1d)
-          val bF = color.map(_.b).getOrElse(1d)
+          val isColorInverted = invertColor.getOrElse(false)
```

Comment:
not super happy with the naming, do we need two names in here, with one for the option and one for the getOrElsed one? Maybe this can be passed here like that already? Then this could always have the same name (invertColor is good, I’d say)